crates/cli/src/model/config_file.rs

```diff
@@ -1,19 +1,28 @@
+use globset::{GlobBuilder, GlobMatcher};
 use std::collections::HashMap;
 use std::fmt;
+use std::path::PathBuf;
 
 use crate::model::serialization::{deserialize_ruleconfigs, deserialize_rulesetconfigs};
 
 use serde;
 use serde::{Deserialize, Serialize};
 
+#[derive(Deserialize, Serialize, Debug, Default, Clone)]
+#[serde(from = "String", into = "String")]
+pub struct PathPattern {
+    pub glob: Option<GlobMatcher>,
+    pub prefix: PathBuf,
+}
+
```

Comment:
Can we document the use of this `struct`? A comment to explain what is it used for?

Reply:
Done.